### ISSUE TITLE
English?

### ISSUE THREAD

**Issue by tobimensch** [emacs-china/Spacemacs-rocks#102]
Hi,

I guess your spacemacs tutorials/book/etc. are excellent.

I would love to  have an English version of it!

Greetings

**Comment by zilongshanren** [comment]
I will discuss with the contributors about how to make an English version. 
And I think an English version would make sense to help the Emacs community to grow.
But most of us are non English native speakers, so we also need help.

Thanks.

**Comment by phoe** [comment]
Hey,

If this is still valid, I'll be willing to help and correct any English text that you might produce.
I'm not a native English speaker myself but I think I know it well enough to be able to help with it.

**Comment by jiegec** [comment]
Yes, thanks very much! Contributions are welcomed via Pull Requests!

**Comment by vipzrx** [comment]
+me
